testplan/testing/multitest/base.py

```diff
@@ -392,6 +392,11 @@ def get_test_context(self):
         ctx = []
         sorted_suites = self.cfg.test_sorter.sorted_testsuites(self.cfg.suites)
 
+        if hasattr(self.cfg, "xfail_tests") and self.cfg.xfail_tests:
```

Comment:
set self.cfg.xfail_tests default value to `{}` ?

Reply:
(i'd like to pretend that) self.cfg is immutable 😄